schema/PACK.xsd

```diff
@@ -17,14 +17,16 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 
-  $Date:        21. Apr 2022
-  $Revision:    1.7.7
+  $Date:        01.06.2022
+  $Revision:    1.7.8
 
   $Project: Schema File for Package Description File Format Specification
 
   Package file name convention <vendor>.<name>.<version>.pack
+  SchemaVersion=1.7.8
 
-  SchemaVersion=1.7.7
+  1. June 2022:
```

Comment:
Version is missing, please keep the format:
```suggestion
  1. June 2022: v1.7.8
```

Reply:
Done